PERFTEST.PS1

```diff
@@ -8,11 +8,11 @@ Param(
     
     [ValidateSet("Default", "Azure", "Detail", "Max")]
     [parameter(Mandatory=$false)] [string] $Config = "Default",
-    [parameter(Mandatory=$false)]  [Array] $ToolList = @("ntttcp", "latte", "cps", "ctstraffic")
+    [parameter(Mandatory=$false)]  [Array] $ToolList = @("ntttcp", "latte", "l4ping", "cps", "ctstraffic")
```

Comment:
nit: use single quotes

Reply:
Copied this from the way it was done in PerfTest before. Do you want me to fix all the old values as well?

Reply:
It's a nit.  go ahead and leave as is.